app/javascript/lib/imperial.js

```diff
@@ -128,6 +129,12 @@ export default class Imperial {
         if (this.auction?.inAuction) {
           Object.assign(this.auction, this.oldAuctionState);
         }
+
+        const currentNation = this.nations.get(this.currentNation);
+        if (currentNation.rondelPosition) {
+          const lastRondelSlot = this.#game.rondel.idToEntity(currentNation.rondelPosition);
```

Comment:
I know `this.#game` was already in this file but I must have missed it before. Previously, my understanding of the instantiation of an `Imperial` was that it was a game itself. Now it looks like a game is one of its properties. Is that right? If that's the case, what is an `Imperial`? (ie, this class, the container of the game) Maybe we should call it `ImperialGameRunner` or something?

Reply:
Yeah, Im actually treating `imperial.js` as more of a coordinator class (and for now have named as such in the unittests)

Haven't gotten around to actually renaming the file yet since there's a lot of other classes that import `imperial.js`, and doing so would incur massive diffs

Next PR?

Reply:
Sure sounds good to me! There's also this issue which I stumbled upon earlier today in one of my games: https://github.com/Thrillberg/imperial/issues/732 I can work on it but if you'd rather, that's fine too.

Reply:
Naw, going to wrap all this up first before opening up another front